### PR TITLE
Updated Creating a basic site tutorial

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Updated steps/screenshots as per v14 release.

TO DO: 

- [ ] Update language variants split view image. Currently, RTE does not display if it's an invariant field. I've raised an issue in the tracker.
- [ ] Update list view on content node image. I've notified the team about the bug in the #bellisma channel.

## Type of suggestion

* [x] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-cms/tutorials/creating-a-basic-website/images/article-main-frontend-v14.png
Comment (jonat123) [inline 14/umbraco-cms/tutorials/creating-a-basic-website/images/article-main-frontend-v14.png:1]: Maybe don't use a picture saying Umbraco 14 Release candidate, unless you are going to update the image after the release 😄

**Review by jonat123 — APPROVED** [review]
@eshanrnh I only have one comment, otherwise it looks good to me :D

File: 14/umbraco-cms/tutorials/creating-a-basic-website/images/article-main-frontend-v14.png
Comment (eshanrnh) [reply 14/umbraco-cms/tutorials/creating-a-basic-website/images/article-main-frontend-v14.png:1]: It's the main article/blog actually that is displaying in the image as part of the tutorial. I'll try changing it to another article and display that in the image :)

**Comment by eshanrnh** [comment]
Thanks for your review, @jonat123 🙌 I've updated the image. Merging it in. 💪

**Pull request merged**